### PR TITLE
Feat: [ADP-3043]Add a tools package and the tools include injectStores function

### PR DESCRIPTION
**Pull request by Arvin-xc**
[QA: None]
Follow: https://compass-tech.atlassian.net/browse/ADP-3043

### CODE REVIEW

File: packages/tools/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "keywords": [
+    "mobx",
+    "devtools"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mobx-devtools/mobx-devtools.git/"
+  },
+  "author": "xichuan.liu@compass.com",
+  "license": "ISC",
```
Comment (hileix) [inline packages/tools/package.json:19]: I think we can use MIT license.

File: packages/tools/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    "@babel/core": "^7.16.0",
+    "@rollup/plugin-typescript": "^8.3.0",
+    "@types/node": "^16.11.9",
+    "rollup": "^2.60.1",
+    "rollup-plugin-typescript": "^1.0.1",
+    "rollup-plugin-uglify": "^6.0.4",
+    "tslib": "^2.3.1"
+  },
+  "dependencies": {
+    "typescript": "^4.5.2"
```
Comment (hileix) [inline packages/tools/package.json:30]: typescript should be placed in `devDependencies`, as typescript will only be used during development.

File: packages/tools/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+declare type Stores = {
+  [storeName: string]: Object;
+};
+
+declare type StoresHook = {
+  stores: Stores;
+};
+declare interface Window {
```
Comment (hileix) [inline packages/tools/index.d.ts:8]: We don't need to write our own `.d.ts` file.

Comment (Arvin-xc) [reply packages/tools/index.d.ts:8]: 👌

File: packages/tools/package.json (hunk 2 above)
Comment (Arvin-xc) [reply packages/tools/package.json:28]: 👌

**Review by hileix — APPROVED** [review]
LGTM!

**Pull request merged**